features/step_definitions/basic_steps.rb

```diff
@@ -31,12 +31,18 @@
 def page_path(path)
   if path == 'Create Article'
     new_article_path
-  elsif path == 'Holger is the best'
-    article_title = Article.find_by(title: path)
-    article_path(article_title)
   elsif path == 'Index'
     root_path
+  elsif (path == 'Holger is the best') || (path == 'Jade loves cookies')
```

Comment:
This one has a potential to grow. Perhaps extract it to a separate method?

Reply:
We tried this but couldn't get it to work. 
maybe we can talk about it tomorrow!